pkgs/build-support/vm/linuxkit-builder/default-key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Maybe give this a scarier name or a comment at the top, explaining where this is used and why you shouldn't be freaking out about a private key in the repo.